Util/src/main/java/io/deephaven/util/mutable/MutableInt.java

```diff
@@ -0,0 +1,68 @@
+//
+// Copyright (c) 2016-2024 Deephaven Data Labs and Patent Pending
+//
+package io.deephaven.util.mutable;
+
+/**
+ * Minimal mutable wrapper for an {@code int} value. Loosely based on
+ * {@code org.apache.commons.lang3.mutable.MutableInt}, but without inheriting from {@link Number}, or providing any
+ * overloads that accept {@code Number} or any boxed types.
+ * <p>
+ * Deliberately does not extend {@code Number}, does not implement {@code toString()}/{@code equals}/{@code hashcode()},
+ * or implement {@code Comparable}.
+ */
+
+public class MutableInt {
+    private int value;
+
+    public MutableInt() {
+
+    }
+
+    public MutableInt(final int value) {
+        this.value = value;
+    }
+
+    public int intValue() {
```

Comment:
Counterargument (no real bias, just want to make sure we don't get grumpy about it):
 * This change will add another 850 lines to the diff. Changing MutableLong.longValue() is another 300 lines.
 * Are you sure you don't want to match AtomicInt/Long with `get()` (and `set()` instead of `setValue()`) instead?

Reply:
I do indeed prefer `get()` and `set()`.

Reply:
Ok, I'll add another commit at the end of this branch changing those.

Reply:
One more counterargument before we proceed: forcing `intValue()` and `longValue()`, while removing the other methods from `Number`, makes it explicit what type you'll be getting out, so what kinds of operations you should be doing with it safely.